standaloneChrome/Dockerfile.txt

```diff
@@ -0,0 +1,13 @@
+MAINTAINER Selion <selion@googlegroups.com>
+
+USER root
+
+#====================================
+# Scripts to run Standalone
+#====================================
+COPY entry_point.sh /opt/selion/entry_point.sh
+RUN chmod +x /opt/selion/entry_point.sh
+
+USER seluser
+
+EXPOSE 4444
```

Comment:
I like the idea of adding `EXPOSE` to all of our Dockerfiles. However, if someone does `SELION_OPTS="-port 1234"` this will have not meaning.